packages/atomic/src/components/common/radio-button.tsx

```diff
@@ -53,12 +53,12 @@ export const RadioButton: FunctionalComponent<RadioButtonProps> = (props) => {
 
   return (
     <input
-      {...attributes}
       type="radio"
       onChange={(e) =>
         (e.currentTarget as HTMLInputElement).checked && props.onChecked?.()
       }
       onMouseDown={onMouseDown}
+      {...attributes}
```

Comment:
Out of curiosity, why are you appending the attributes after the other ones? Unless I am mistaken this change has no effect.

Reply:
good question. Stencil was giving me warnings saying that a certain prop needed to be after others so I did this and there no more warnings

Reply:
The change would have been breaking if one of the 3 attributes declared on the element were also part of the `attributes` object. I wonder if the same warning would have been raised.

Reply:
It's much more logical to have the spread attributes after the others indeed. I did a quick scan of everywhere this radio button is used, and there should not be any breaking changes 👍